src/components/coursesSection/courses/Courses.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import Course from "../course/Course";
+import coursesStyle from "./Courses.module.css";
+
+function Courses() {
```

Comment:
this component should accept the courses as a prop.

Reply:
Done